### ISSUE TITLE
Getting issue when adding multiple Key and its value.

### ISSUE THREAD

**Issue by ankushgoyal27** [sjhoeksma/cordova-plugin-keychain-touch-id#14]
Objective: Need to add multiple username and its password

Error: Different keys are available but their passwords are replaced with the latest one. It means in the keychain multiple keys are available but all has same passwords.

Any suggestion.

**Comment by mhanke** [comment]
Have you found a solution for this?

**Comment by ankushgoyal27** [comment]
@mhanke Yes you need to update the iOS plugin. Because currently it only supports for user, try to modify and make it compatible for multiple users.
